automation/check-patch.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -xe
+
+install_deps() {
+    pip install pytest
+}
+
+main() {
+    install_deps
+    trap 'make cluster-down' EXIT
+    make cluster-up
```

Comment:
We should do `make build` too, but for that we need to install golang.
How should we proceed? Do you want to spend time debugging it or do we just merge empty main? I can fix it later.

Reply:
Which version of golang the build needs?

Reply:
Not sure, I use 1.9.7